core/lib/instrument_api.h

```diff
@@ -796,8 +796,7 @@ DR_API
  * Clients may want to avoid touching resources shared between processes,
  * like files, from the post-fork execution of the callback. The post-fork
  * version of the callback can be recognized by dr_get_process_id()
- * returning a different value than it returned during the corresponding
- * thread init event.
+ * returning a different value than dr_get_process_id_from_drcontext(drcontext).
```

Comment:
Does Doxygen turn it into a link if it has a parameter name?  It definitely makes links for empty parens which is why we always have those for function references in Doxygen comments.  Maybe quickly check your api/docs/html/ output to confirm; if no link then remove the param name.

Reply:
The parameter name is removed.